rosbag2_tests/CMakeLists.txt

```diff
@@ -22,10 +22,12 @@ find_package(ament_index_cpp REQUIRED)
 
 find_package(PkgConfig)
 if(PKG_CONFIG_FOUND)
-  find_package(ros1_bridge REQUIRED)
-  include(${ros1_bridge_DIR}/find_ros1_interface_packages.cmake)
-  include(${ros1_bridge_DIR}/find_ros1_package.cmake)
-  find_ros1_package(roscpp)
+  find_package(ros1_bridge)
+  if(ros1_bridge_DIR)
+    include(${ros1_bridge_DIR}/find_ros1_interface_packages.cmake)
+    include(${ros1_bridge_DIR}/find_ros1_package.cmake)
```

Comment:
It would be better to update the `ros1_bridge` package to include these files when being found.